scripts/ci/create-binaries.sh

```diff
@@ -21,6 +21,7 @@ cd ${GOPATH}/src/github.com/skydive-project/skydive
 
 echo "--- BINARIES ---"
 make static WITH_EBPF=true
+make contribs
```

Comment:
The contribs executables built this way are not static.

Reply:
removed ('make static' already invokes static build of exporters)